src/Vulkan/VkDebug.h

```diff
@@ -10,4 +11,20 @@ VKAPI_ATTR VkBool32 VKAPI_CALL debugCallback(::vk::DebugUtilsMessageSeverityFlag
                                              const ::vk::DebugUtilsMessengerCallbackDataEXT* pCallbackData,
                                              void* pUserData);
 
-}
\ No newline at end of file
+#if defined(VEX_DEBUG) or defined(VEX_DEVELOPMENT)
```

Comment:
FYI they are always defined, so it should just be `#if VEX_DEBUG or VEX_DEVELOPMENT`. (or `#if !VEX_SHIPPING` to match DX12's debug naming).